compose/container.py

```diff
@@ -172,6 +172,19 @@ def has_api_logs(self):
         log_type = self.log_driver
         return not log_type or log_type in ('json-file', 'journald')
 
+    @property
+    def create_health_status_string(self):
```

Comment:
If a method is marked with an `@property` decorator, it should have a property name instead of an action name. To be consistent with other similar properties, maybe rename this to `human_readable_health_status`?